std/format.d

```diff
@@ -3251,6 +3251,29 @@ if (is(AssocArrayTypeOf!T) && !is(T == enum) && !hasToString!(T, Char))
 
 @safe unittest
 {
+    // Bug #17269. Behavior similar to `struct A { Nullable!string B; }`
```

Comment:
Why not use a struct with `Nullable!string`?

Reply:
If I use a Nullable!string, it causes an [assert failure on another unittest block](https://github.com/dlang/phobos/blob/master/std/format.d#L4035)
```
expected = `void delegate() @system`, result = `void delegate()`
```

I don't really understand why, so I wrote a custom struct to:
- Show the `alias this` and the implicit conversion problem
- Do not depend on Nullable if it is changed in the future

Reply:
Huh? `Nullable!string` works without your PR: https://run.dlang.io/is/Jjzpvo

Reply:
Could you provide a test case where it fails and maybe open another bug report?

Reply:
you need to put the Nullable!string inside a struct to see it fail:
```d
struct NullableStr {
   Nullable!string value;
}
NullableStr f;
format("%s", f).writeln;
```